modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -218,7 +229,8 @@ spec:
 <11> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
 <12> Specify a snapshot location, unless you use CSI snapshots or Restic to back up PVs.
 <13> The snapshot location must be in the same region as the PVs.
-<14> Google workload identity federation supports internal image backup. Set this field to `false` if you do not want to use image backup.
+<14> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location.
+<15> Google workload identity federation supports internal image backup. Set this field to `false` if you do not want to use image backup.
```

Comment:
Noticed this only because I have written about this topic too - afaik it is initial caps ("[Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation)"). I don't think you need to change this instance, since I see it is also used elsewhere like you have it here. It might be nice to change them all at some point to match how Google describes their feature.